app.rb

```diff
@@ -0,0 +1,32 @@
+require 'sinatra'
+require 'erb'
+require 'pry-byebug'
+require './helpers/weather_scraper.rb'
+require './helpers/craigslist_scraper.rb'
+
```

Comment:
This is interesting. So it worked properly just with require helpers? I'm guessing you separated weather and scraper into separate modules; or if you just had them as classes maybe you don't have to do the `helpers weather_scraper` syntax after. If you see in my code I both required it and did `helpers SchwaddyHelper` after.